packages/react/src/ActionBar/ActionBar.tsx

```diff
@@ -217,16 +217,10 @@ export const ActionBar: React.FC<React.PropsWithChildren<ActionBarProps>> = prop
                     if (menuItem.type === ActionList.Divider) {
                       return <ActionList.Divider key={index} />
                     } else {
-                      const {
-                        children: menuItemChildren,
-                        onClick,
-                        icon: Icon,
-                        'aria-label': ariaLabel,
-                        disabled,
-                      } = menuItem.props
+                      const {onClick, icon: Icon, 'aria-label': ariaLabel, disabled} = menuItem.props
                       return (
                         <ActionList.Item
-                          key={menuItemChildren}
+                          key={ariaLabel}
```

Comment:
Why not make it related to `index` ? Would index be changing between renders somehow?